test.properties

```diff
@@ -4134,24 +4134,23 @@
         modules-unit-jdk8
 
     test.batch.run.property.query[functional-tomcat90-mysql57-jdk8][frontend]=\
-        (portal.upstream == true) AND \
         (\
-            (testray.component.names ~ "Cadmin") OR \
-            (testray.component.names ~ "Frontend Dataset") OR \
-            (testray.component.names ~ "Remote Apps") OR \
-            (testray.component.names ~ "Theme") OR \
-            (testray.component.names ~ "Translation Manager") OR \
-            (testray.component.names ~ "WYSIWYG") OR \
-            (testray.component.names ~ "Walkthrough") OR \
```

Comment:
Just wondering these removed components no longer belong to frontend anymore?

Reply:
The component.names can be applied on tests that are not owned by FI team, they don't necessarily belong to FI. ie: On BlogsAdmin#CanAddEntryInAdmin it has `testray.component.names ~ "WYSIWYG"`. Test still owned by Lima team, but I placed WYSIWYG component for `ci:test:relevant` purposes to target usecase tests for ckeditor module.

On Testray, I believe it uses  `testray.main.component.name` to filter by `Team` and would better define which tests belong to frontend infra.

Reply:
Good to know, thanks so much for the explanation!